### DISCUSSION TITLE
part-ii/2018-4/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#53] [Question comments]
Part II 2018 Algebraic Topology: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2018-4/

**Comment by artie2000** [comment]
Anyone got any ideas for the cover to take? What a strange question..

**Comment by ArthurConmy** [comment]
Does this work? For initial work, $\pi_1(X_n) = \langle a b_1, ... , b_n | a^2 \rangle $ and draw $\hat{X}_n$:

![](https://i.imgur.com/vqjDZe5.png)

then by contracting the two spheres, $H_1(\hat{X}_n) = \mathbb{ Z}^{ 4n }$. Since $H_1$ is the abelianization of $\pi_1$, and the covering is normal, if $X_n$ is homotopic to a closed compact surface, then it has an index 4 subgroup that abelianizes to $\mathbb{ Z}^{ 4n }$. As some side work, as $\pi_1(X_n)$ abelianizes to $\mathbb{ Z}^n \times C_2$ which means that the compact surface we're homotopic to can only be $E_{n-1}$. 

But index 4 normal subgroups of $\pi_1(X_n,x_0)$ are defined by surjective maps to $C_4$, and these can be described by where the generators $a, b_1, ... , b_n$ get mapped to ($a$ must map to 0 or 2, all other $b_i$ may map to $0, 1, 2$ or $3 \in C_4$ so long as the final map is surjective). All such subgroups abelianize to something with a $\mathbb{ Z}^n$ (possibly with $\times C_2$), and none have the $\mathbb{ Z}^{4n}$.

**Reply by artie2000** [reply]
I'm not sure that we are allowed to use the abelianisation fact 😦.

You can use the fact that covers of surfaces are surfaces to obtain a more direct contradiction using this cover: $H_2(\hat{X}_n)=\mathbb{Z}^2$, which is not the same for any surface.
